.github/workflows/cibuild.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: ["3.10", "3.11", "3.12"]
```

Comment:
I'm not sure Python 3.12 is ready for prime time. Pretty sure that failure has to do with versioneer.

Reply:
But wait!

Reply:
Looks like the bug has already been fixed in versioneer. So maybe instead of dropping 3.12 from the CI build matrix, I'll try updating it using these instructions.